src/style.css

```diff
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.page {
+  margin: 0;
+}
+
+.card {
+  padding: 32px 16px 16px;
+}
+
+.page__product {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  max-width: 160px;
+  max-height: 134px;
+  margin-bottom: 40px;
+}
```

Comment:
Add extra paddings (left and right) 3px for image (check in figma)
Then try to retry the tests